pxr/usd/plugin/usdDraco/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+set(PXR_PREFIX pxr/usd)
+set(PXR_PACKAGE usdDraco)
+
+pxr_plugin(${PXR_PACKAGE}
+    LIBRARIES
+        tf
+        gf
+        sdf
+        usd
+        usdGeom
+        draco
```

Comment:
I think typically we like to refer to external dependencies via a build variable, in case the lib name changes in the future?

Reply:
Done.